docs/topics/i18n/translation.txt

```diff
@@ -875,6 +876,7 @@ There are also simple filters available for convenience:
 
 * ``{{ LANGUAGE_CODE|language_name }}`` ("German")
 * ``{{ LANGUAGE_CODE|language_name_local }}`` ("Deutsch")
+* ``{{ LANGUAGE_CODE|language_name_translated }}`` ("německy", when active language is Czech)
```

Comment:
Need `.. versionadded:: 1.9` annotations to describe the newly added features and also a mention in the release notes.